.editorconfig

```diff
@@ -1,7 +1,22 @@
 # top-most EditorConfig file
 root = true
 
+# Visual Studio add on:  https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328
+# Notepad++ plugin:  https://sourceforge.net/projects/editorconfig/files/EditorConfig-Notepad%2B%2B-Plugin/
+# For other code editors:  http://editorconfig.org/#download
+
 [*]
 indent_style = space
 indent_size = 4
 end_of_line = lf
+
+[*.lua]
+trim_trailing_whitespace = true
+
+[*.{cpp,h}]
+trim_trailing_whitespace = true
+indent_brace_style = Allman # This is unfortunately ignored in the Visual Studio add on
```

Comment:
Its what the style of indent and bracing DSP uses is actually called. Named for the guy that started it.